Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/setup.py

```diff
@@ -102,10 +102,11 @@ def github(cls, args, repository, additional_setup=None, remote=None, team=None,
                 return result
 
         if repository.owner == username or args.defaults or Terminal.choose(
-            "Create a private fork of '{}' belonging to '{}'".format(forked_name, username),
-            default='Yes',
+            "Create a private fork of '{}/{}' named '{}' belonging to '{}'".format(
+                repository.owner, repository.name, forked_name, username
+            ), default='Yes',
         ) == 'No':
-            log.info("Continuing without forking '{}'".format(forked_name))
+            log.info("Continuing without forking '{}/{}'".format(repository.owner, repository.name))
```

Comment:
would love to see Python 3 here in this codebase when possible. f-strings can make it more readable.